src/readability_classifier/encoders/dataset_encoder.py

```diff
@@ -56,9 +56,11 @@ def encode_dataset(self, unencoded_dataset: list[dict]) -> ReadabilityDataset:
         bert_dataset = self.bert_encoder.encode_dataset(unencoded_dataset)
         image_dataset = self.visual_encoder.encode_dataset(unencoded_dataset)
 
-        # Normalize the scores
-        scores = [sample["score"] for sample in unencoded_dataset]
-        encoded_scores = self._encode_scores_class(scores)
+        # Normalize the scores if they exist
+        encoded_scores = ["" for _ in range(len(matrix_dataset))]
+        if "scores" in unencoded_dataset[0]:
```

Comment:
Singular:
if "score" in unencoded_dataset[0]:

Reply:
@LuKrO2011 Fixed with 7cc9b639b9f431d53429ce78914574a9591e3ef2